### PR TITLE
add/improve some icons

### PR DESCRIPTION
**Pull request by frontendschlampe**
- improve icon modules.svg
- add icons for https://github.com/contao/core-bundle/issues/1202
- add icon for https://github.com/contao/core-bundle/pull/1545

### CODE REVIEW

File: src/Resources/contao/themes/flexible/icons/modules.svg [hunk 1]
```diff
@@ -1 +1 @@
...
...-.13-.93l.9-.54a4.79 4.79 0 0 0-.45-1l-1 .26a3.78 3.78 0 0 0-1-1l.28-1A4.68 4.68 0 0 0 7.71 5l-.55.91a3.57 3.57 0 0 0-.8-.09 4.43 4.43 0 0 0-.9.11L5 5a5.22 5.22 0 0 0-1 .4l.29 1a4 4 0 0 0-1.15 1.11l-1.07-.2a4.56 4.56 0 0 0-.42 1l.91.54a3.86 3.86 0 0 0-.1.84 3.75 3.75 0 0 0 .09.82l-.9.54a4.49 4.49 0 0 0 .42 1l1-.28A3.81 3.81 0 0 0 4.19 13l-.28 1a4.86 4.86 0 0 0 1 .43l.55-.89a3.84 3.84 0 0 0 .91.12 4.27 4.27 0 0 0 .81-.09l.53.89a5.13 5.13 0 0 0 1-.41l-.27-1a4 4 0 0 0 1.19-1.17l1 .28a5.45 5.45 0 0 0 .42-1l-.86-.66a3.67 3.67 0 0 0 .07-.79zM6.35 11.4a1.7 1.7 0 1 1 1.7-1.7 1.7 1.7 0 0 1-1.7 1.7zm8.07-7.09l-.64-.12a2.44 2.44 0 0 0-.27-.8l.42-.53a3.07 3.07 0 0 0-.41-.47l-.62.31a2.11 2.11 0 0 0-.8-.38l-.1-.67a3.13 3.13 0 0 0-.44 0h-.24l-.11.65a2.13 2.13 0 0 0-.82.3l-.47-.47a3.42 3.42 0 0 0-.49.44l.32.62a2 2 0 0 0-.35.72L8.71 4a2.73 2.73 0 0 0-.05.5 1.36 1.36 0 0 0 0 .2l.64.1a2.28 2.28 0 0 0 .27.81l-.46.46a3 3 0 0 0 .41.51l.62-.31a2.18 2.18 0 0 0 .82.41v.66a3.31 3.31 0 0 0 .55 0h.14l.1-.65a2.22 2.22 0 0 0 .88-.28l.53.43a3.39 3.39 0 0 0 .45-.4l-.35-.57a2.24 2.24 0 0 0 .42-.81l.71-.06a2.76 2.76 0 0 0 0-.51 1.11 1.11 0 0 0 .03-.18zm-2.87 1.15a1 1 0 1 1 1-1 1 1 0 0 1-1 1z" fill="url(#a)"/></svg>
```
Comment (leofeyer) [inline src/Resources/contao/themes/flexible/icons/modules.svg]: `width="5.64mm"`?

**Comment by leofeyer** [comment]
Why did you change the existing `modules.svg` icon?

**Comment by frontendschlampe** [comment]
> Why did you change the existing `modules.svg` icon?

The circle was not a circle. It was a circle with bumps. That's why I rework it. With the mm should not happen. I will update my PR.

File: src/Resources/contao/themes/flexible/icons/modules_.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...-.13-.93l.9-.54a4.79 4.79 0 0 0-.45-1l-1 .26a3.78 3.78 0 0 0-1-1l.28-1A4.68 4.68 0 0 0 7.71 5l-.55.91a3.57 3.57 0 0 0-.8-.09 4.43 4.43 0 0 0-.9.11L5 5a5.22 5.22 0 0 0-1 .4l.29 1a4 4 0 0 0-1.15 1.11l-1.07-.2a4.56 4.56 0 0 0-.42 1l.91.54a3.86 3.86 0 0 0-.1.84 3.75 3.75 0 0 0 .09.82l-.9.54a4.49 4.49 0 0 0 .42 1l1-.28A3.81 3.81 0 0 0 4.19 13l-.28 1a4.86 4.86 0 0 0 1 .43l.55-.89a3.84 3.84 0 0 0 .91.12 4.27 4.27 0 0 0 .81-.09l.53.89a5.13 5.13 0 0 0 1-.41l-.27-1a4 4 0 0 0 1.19-1.17l1 .28a5.45 5.45 0 0 0 .42-1l-.86-.66a3.67 3.67 0 0 0 .07-.79zM6.35 11.4a1.7 1.7 0 1 1 1.7-1.7 1.7 1.7 0 0 1-1.7 1.7zm8.07-7.09l-.64-.12a2.44 2.44 0 0 0-.27-.8l.42-.53a3.07 3.07 0 0 0-.41-.47l-.62.31a2.11 2.11 0 0 0-.8-.38l-.1-.67a3.13 3.13 0 0 0-.44 0h-.24l-.11.65a2.13 2.13 0 0 0-.82.3l-.47-.47a3.42 3.42 0 0 0-.49.44l.32.62a2 2 0 0 0-.35.72L8.71 4a2.73 2.73 0 0 0-.05.5 1.36 1.36 0 0 0 0 .2l.64.1a2.28 2.28 0 0 0 .27.81l-.46.46a3 3 0 0 0 .41.51l.62-.31a2.18 2.18 0 0 0 .82.41v.66a3.31 3.31 0 0 0 .55 0h.14l.1-.65a2.22 2.22 0 0 0 .88-.28l.53.43a3.39 3.39 0 0 0 .45-.4l-.35-.57a2.24 2.24 0 0 0 .42-.81l.71-.06a2.76 2.76 0 0 0 0-.51 1.11 1.11 0 0 0 .03-.18zm-2.87 1.15a1 1 0 1 1 1-1 1 1 0 0 1-1 1z" fill="url(#a)"/></svg>
```
Comment (leofeyer) [inline src/Resources/contao/themes/flexible/icons/modules_.svg]: Width and height are still `5.64mm` here.

File: src/Resources/contao/themes/flexible/icons/sizes_.svg [hunk 3]
```diff
@@ -0,0 +1 @@
...z" fill="url(#e)"/><path d="M15.54.46v10H3.66v-10h11.88M16 0H3.2v11H16z" fill="#fff"/><path d="M1.83 2.7h12.34v10.51H1.83z" fill="url(#f)"/><circle cx="4.8" cy="6.13" r="1.87" fill="url(#g)"/><path d="M7.33 12.05l1.3-.46 1.06-1.85 1.25.81.73-.18 1.54-1.26h1V8.3l-1.4-.81-1.35 1-1.68-2-2.36 4-1.73-.87-2.77 2.47 3-.56z" fill="url(#h)"/><path d="M14.17 13.26V9.15h-1l-1.54 1.26-.73.18-1.21-.85-1.06 1.85-1.3.46-1.45-.52-3 .56-1.14 1.05z" fill="url(#i)"/><path d="M14.17 13.26V9.15h-1l-1.54 1.26-.73.18-1.21-.85-1.06 1.85-1.3.46-1.45-.52-3 .56-1.14 1.05z" fill="url(#j)"/><path d="M13.94 3v10H2.06V3h11.88m.46-.46H1.6v11h12.8z" fill="#fff"/><path d="M.23 5.21h12.34v10.52H.23z" fill="url(#k)"/><circle cx="3.2" cy="8.64" r="1.87" fill="url(#l)"/><path d="M5.73 14.56L7 14.1l1.06-1.85 1.25.81.73-.17 1.54-1.27 1 .05v-.86l-1.4-.8-1.36.99-1.68-2-2.32 4-1.73-.86-2.77 2.46 3-.56z" fill="url(#m)"/><path d="M12.57 15.77v-4.1l-1-.05-1.54 1.27-.73.17-1.25-.81L7 14.1l-1.3.46L4.28 14l-3 .56-1.14 1.1z" fill="url(#n)"/><path d="M12.57 15.77v-4.1l-1-.05-1.54 1.27-.73.17-1.25-.81L7 14.1l-1.3.46L4.28 14l-3 .56-1.14 1.1z" fill="url(#o)"/><path d="M12.34 5.49v10H.46v-10h11.88M12.8 5H0v11h12.8z" fill="#fff"/></svg>
```
Comment (leofeyer) [inline src/Resources/contao/themes/flexible/icons/sizes_.svg]: Width and height are still `5.64mm` here.

File: src/Resources/contao/themes/flexible/icons/theme_export_.svg [hunk 4]
```diff
@@ -0,0 +1 @@
...-.45.16-.5-.18-1 .19-.41.37z" fill="url(#l)"/><path d="M7.15 13.2v-1.42h-.34l-.53.44-.28.04-.4-.26-.37.64-.45.16-.5-.18-1 .19-.41.37z" fill="url(#m)"/></g><path d="M11.16 8.52L15.87 1a.61.61 0 0 0-.11-.86.62.62 0 0 0-.87.11l-5.41 7 1.68 1.28z" fill="url(#n)"/><path d="M10.82 8.62l-1.35-1-1.56 2.57.66.5z" fill="url(#o)"/><path d="M7.22 10.58a1 1 0 0 0-.49.82c0 .6-.35.61-.79.71a1.34 1.34 0 0 1-.94-.28s1.08 1.17 2 .94a1.78 1.78 0 0 0 1.37-1.14 1.33 1.33 0 0 0 0-.53s-.38-1-1.15-.52z" fill="url(#p)"/><path d="M9.8 15.61a1 1 0 0 1-1-1V12v-.36H7.08a.33.33 0 0 1-.3-.2c-.08-.18 0-.34.45-.84l3-3.15a1 1 0 0 1 1.41 0l3 3.15c.46.48.5.66.43.83a.34.34 0 0 1-.31.21h-1.6a3.21 3.21 0 0 0 0 .36v2.6a1 1 0 0 1-1 1z" fill="url(#q)"/><path d="M11 7.42a.69.69 0 0 1 .48.2l3 3.16c.26.27.42.49.35.49h-1.91c-.05 0-.1.3-.1.68v2.65a.68.68 0 0 1-.68.69H9.8a.69.69 0 0 1-.68-.69V12a1.57 1.57 0 0 0-.12-.73H7s.12-.22.38-.49l3-3.16a.67.67 0 0 1 .47-.2m0-.64a1.28 1.28 0 0 0-.93.4L7 10.33c-.13.14-.25.27-.33.37a.79.79 0 0 0-.18.81.64.64 0 0 0 .59.4h1.4v2.69a1.33 1.33 0 0 0 1.32 1.33h2.34a1.33 1.33 0 0 0 1.32-1.33v-2.69h1.38a.66.66 0 0 0 .6-.4.81.81 0 0 0-.16-.8l-.28-.38-3-3.15a1.29 1.29 0 0 0-.94-.4z" fill="#fff"/></svg>
```
Comment (leofeyer) [inline src/Resources/contao/themes/flexible/icons/theme_export_.svg]: Width and height are still `5.64mm` here.

**Comment by leofeyer** [comment]
Thanks a lot @frontendschlampe.

**Comment by bytehead** [comment]
🎉

**Pull request merged**